docs-src/content/getting-started/guide.md

```diff
@@ -4,453 +4,121 @@
-#   completion  Generate the autocompletion script
-#   help        Help about any command
-#   operator    Manage admin operations
-#   policy      Manage policies
-#   secret      Manage secrets
-#
-# Flags:
-#  -h, --help   help for spike
-# 
-# Use "spike [command] --help" for help.
-```
-
-Although the `make start` script is convenient, it might be useful
-to run the components individually to understand the process better and
-debug any issues that might arise.
-
-The following sections will guide you through the individual steps.
-
-> **CLI Reference**
->
-> Since the **SPIKE CLI** is a work in progress and highly in flux, the best
-> way to get the most up-to-date information is to run `spike --help` or
-> `spike [command] --help` to learn about the available commands and flags.
->
-> In addition, you can [check out the demo recordings][demo] to see the CLI in
-> action.
-
-[demo]: @/community/presentations.md
-
-## Start SPIRE Server
+helm repo add spiffe https://spiffe.github.io/helm-charts-hardened
+helm repo update
 
-Start the SPIRE Server:
-
-```bash
-cd $WORKSPACE/spike
-./hack/spire-server-start.sh
-```
-
-## Creating Registration Entries
-
-The following script will create registration entries for the SPIKE components:
-
-```bash
-cd $WORKSPACE/spike
-./hack/spire-server-entry-spike-register.sh
-```
-
-## Start SPIRE Agent
-
-Start the SPIRE Agent:
-
-```bash
-cd $WORKSPACE/spike
-./hack/spire-agent-start.sh
-```
-
-## Start SPIKE Components
-
-Then start **SPIKE** components:
-
-Make sure you started the following binaries, each runs on a specific terminal
-window.
-
-Start the workloads:
-
-```bash
-# Optional: Increase the log level to debug:
-export SPIKE_SYSTEM_LOG_LEVEL=debug
-
-cd $WORKSPACE/spike
-
-# Start SPIKE Nexus in one terminal.
-./hack/start-nexus.sh
-
-# Start SPIKE Keepers in separate terminals.
-./hack/start-keeper-1.sh
-./hack/start-keeper-2.sh
-./hack/start-keeper-3.sh
-```
-
-Here is how one of these **SPIKE Keeper** startup scripts looks like:
-
-```bash
-# ./hack/start-keeper-1.sh
-SPIKE_KEEPER_TLS_PORT=':8443' \
-./keeper
-````
-
-And here is how **SPIKE Nexus** startup script looks like:
-
-```bash
-# ./hack/start-nexus.sh
-SPIKE_NEXUS_KEEPER_PEERS='https://localhost:8443,\
-https://localhost:8543,https://localhost:8643'
-./nexus
-```
-
-> **Sequential SPIKE Keeper IDs**
-> 
-> The mapping in `SPIKE_NEXUS_KEEPER_PEERS` should start from `"1"` 
-> and increase monotonically without any gaps in the sequence as shown
-> in the sample code above. This is because of the way SPIKE Nexus internally
-> computes and distributes the Shamir Shards. Not following this sequence
-> will lead to errors---We may improve this behavior in the future and make
-> it more flexible.
-
-## Using SPIKE Pilot
-
-Define an alias to **SPIKE** Pilot:
-
-```bash
-# ~/.bashrc
-
-# path to the SPIKE Pilot binary (`spike`)
-alias spike=$WORKSPACE/spike/spike
+helm upgrade --install -n spire-mgmt spire-crds spire-crds \
```

Comment:
Since this is a demo deployment, might just drop `-n spire-mgmt` and `--create-namespace` everywhere from here and beyond. would simplify the example.